ledger/participant-state/kvutils/tools/engine-benchmark/src/benchmark/scala/ledger/participant/state/kvutils/tools/engine/benchmark/Replay.scala

```diff
@@ -95,7 +95,7 @@ class Replay {
   def bench(): Unit = {
     val result = engine
       .replay(
-        benchmark.transaction.singleSubmitterOrThrow(),
+        Set(benchmark.transaction.singleSubmitterOrThrow()),
```

Comment:
```suggestion
        benchmark.transaction.actAs.toSet,
```

`singleSubmitterOrThrow` is a placeholder used in all places that can't yet handle multiple submitters.

Reply:
I might have misinterpreted the comment from the `singleSubmitterOrThrow()` method:
```
  // Note: this method will be removed when the entire kvutils code base
  // supports multi-party submissions
```
I thought that we have to still wait before removing that.
I will change the PR according to your comment.

Reply:
`actAs` was not available - I've used `submitters` instead.